common/rpctypes/types.go

```diff
@@ -78,12 +78,11 @@ type TakeOfferRequest struct {
 
 // MakeOfferRequest ...
 type MakeOfferRequest struct {
-	MinAmount       *apd.Decimal        `json:"minAmount" validate:"required"`
-	MaxAmount       *apd.Decimal        `json:"maxAmount" validate:"required"`
-	ExchangeRate    *coins.ExchangeRate `json:"exchangeRate" validate:"required"`
-	EthAsset        types.EthAsset      `json:"ethAsset,omitempty"`
-	RelayerEndpoint string              `json:"relayerEndpoint,omitempty"`
```

Comment:
I removed support for non-p2p relayers. In this PR, Bob just uses relayers in the order of discovery, but it might make sense to first request from Alice.